HW03/veaceslav_irizanov/veaceslav_irizanov_hw_03_t_03.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require 'time'
+
+class Homework3
+  def task3(log)
+    final_array = array_of_duration(time_line(total_rows(log)))
+    final_array.size == 1 ? final_array[0] : final_array
+  end
+
+  private
+
+  def total_rows(log)
+    log.split("\n").select { _1.include?('Calling core with action') }
+  end
+
+  def time_line(lines)
+    lines.collect { |line| Time.parse(line) }
+  end
+
+  def array_of_duration(array)
```

Comment:
What array comes to us? 
maybe it makes sense to choose a more appropriate naming?

Reply:
which kind of naming?